### DISCUSSION TITLE
【検討】Mini＆Playground検証企画

### DISCUSSION THREAD

**Discussion by toki-sor1** [snext1220/stext#261] [Ideas]
**【検討】Playground検証企画**(#12) に続くIssueです。
このIssueでは、プレグラ100祭以外に、新しく「ミニミニ30」シナリオの話題も扱います。
どうぞご利用ください。
***
#### 2020年 5月～
- 【プレグラ】Vol.4（5月2日公開済）：ちゃなさん（不思議のソーサリアン）

　※以降、シナリオは隔月連載になります

#### 2020年 6月～
- シナリオ公開なし

#### 2020年 7月～
- シナリオ公開なし
- 【SN2020】Next Scenario 2020（7月13日公開済）：サルバさん

#### 2020年 8月～
- 【プレグラ】Vol.5（8月1日公開予定）：鏡読みさん（たまには日の光を浴びよう）

#### 2020年 9月～
- シナリオ公開なし

#### 2020年10月～
- 【プレグラ】Vol.6（10月26日公開済）：ちゃなさん（ソーサリアベンジャーズ）

#### 2020年11月～
- 【プレグラ】Vol.7（11月or来年公開予定？）：ときのじ（ハクジャ伝説）

***

**Comment by snext1220** [comment]
新規Issueありがとうございます！
まずは8月のminiを進めていければと思いますが、その他にも、現時点では以下のようなものを検討しております（隗より始めよ、で挙げてみる）。

+ ある魔術師の大失敗 or 灰色のダンジョン
    + チュートシナ
    + 天神もありかと思いつつ、チュートはダンジョンものの方がよいか
+ 七星の名のもとに
    + ドラモード外伝
+ 剣と魔法の都ペンタウァ
    + シナコン移植作品
    + オーソドックスな作品になりそうなので、シカケの多いシナの中で埋もれそうかと悩み中
+ 新・アマゾンの剣
    + パーティ制
    + 3シナ同時進行もの。初期から検討しているものの煮詰まらず；
+ 嗚呼、ジョセフィーヌよ、何処へ行く
    + ミニミニ30の時に検討していたもの。気が向いたら

ある程度進められそうであれば、stext.xmlにどんどん登録して誘い水にしていければ、などとも考えています（ミニミニ～もそんな感じでしたが、大丈夫ですかね？？）。

> ＃私もテーマは2つほどあり、片方はゲームの中にゲームを展開するような内容で、もう片方が夏向きの内容なのですが、まだふわっとした状態ですね；

おぉ、楽しみです！夏向きということで、8月枠は全然お譲りできますので、必要であればお声がけください～

**Reply by snext1220** [reply]
黒船が一区切りついたところで、今後の予定をザクッとまとめておきます（予定は未定ですが、まずは備忘録を兼ねて）。早ければ一作は7月（ときのじさんの幕末後）にでも出せればよいなぁと考えています。

+ 新・天の神々たち
    + 以前出ていたチュートリアルシナが手つかずだったのですが、旧シナコン作の移植として復活できればと。
+ その他シナコン作品移植
    + その他にも「脱出」「Ys」「時の封土」なども徐々に移植していければと。
+ ミニミニソーサリアン Text
    + 以前予告していたものです。こちらは他をやりながらネタをためて
+ それいけ！ドトーのトライアスロン（第2回）
    + 原作ネタ。
    + 職業シスを駆使したネタものにはなりそうですが、まだ思い付きレベル
+ 昔話ソーサリアン／はじまりのゼロ／宇宙からの訪問者
    + 早いですが、ポスト幕末の連作ネタとして挙げておきます。
    + 昔話は以前に上様さんから提案のあったもの（https://sorcerian.hateblo.jp/entries/2014/03/17 ）で、テーマの知名度、書きやすさ的にはよさそう
    + はじまりのゼロは0401後の祭り企画のものです（https://sorcerian.hateblo.jp/entries/2013/04/02 ）。旧版ではNextシナをベースにしていましたが、原作シナベース＆STextシナを交えてできればよいなと
     + 宇宙からの～は、ドラモード／幕末ときたので、原作とのリンクという意味だけです（ノーアイデアですが、STextにないSFネタをできればなどと）

**Comment by snext1220** [comment]
進捗していない進捗報告です；
システム改定を優先させていたため、なかなかシナリオ作成に取り掛かれずにいますが、流石に隔月スパンを前提にしても10月は死守しなければと思っております。頑張ってまいります...

そんな中で恐縮ですが、皆さまのシナリオ状況は如何でしょうか。
ポストPg100、幕末等、話題が途切れてから時間も経ってしまったので、伺い方々、一応スレ上げしておきます～^^;

**Comment by toki-sor1** [comment]
お返事が大変遅くなり、申し訳ありません…！；；
ご返信、そして進捗についてもありがとうございました…！

> ある程度進められそうであれば、stext.xmlにどんどん登録して誘い水にしていければ、などとも考えています（ミニミニ～もそんな感じでしたが、大丈夫ですかね？？）。

はい、ミニミニのようにまずは予定としてどんどん登録、良いと思います！

> 流石に隔月スパンを前提にしても10月は死守しなければと思っております。頑張ってまいります...

> そんな中で恐縮ですが、皆さまのシナリオ状況は如何でしょうか。

ご報告が遅くなり申し訳ありません；；
シナリオを幾つか書きかけで紆余曲折し、ようやく和風シナリオにしようと決まり、現在執筆中です；
おそらく10月中には書き上げられるかなぁというところまで大筋の流れはできているので、このまま頑張りたいと思います～…！

多分、ところどころ変わるかもしれませんが、大体は下記のような始まり方のノベルみの強めな中編（？）シナリオです。
***
### ハクジャ伝説　七星御伽草子　一ノ章（仮）
#### Scene 0
　とっぷりと日が暮れて、夜の色に染め上げられた山々は影絵に姿を変え、夜空の下方に並んでいる。
　数え切れぬほどの星が瞬く銀鼠色の僅かな雲が穏やかな風に流れ、煌々と輝く蒼き月の上をなぞっていった。

　『君』は松明片手に、疲れてやや重い足取りで田んぼに囲まれた夜道を歩いている。
　人が二人並んで歩ける程度の道幅だが、君はど真ん中を延々と歩いているのだ。
　誰かとすれ違わないものかと多少の望みは抱いていたが、一向にその気配はない。
　雑草がまばらに生えたデコボコの畦道に人影はなく、君が黙々と歩く音と蛙達の大合唱が聞こえる。ただそれだけである。

　此処は神秘の国『JAPAN』。
　ソーサリアンである君の祖国ペンタウァから遥か遠く離れた東洋の島国である。
　過去に他のソーサリアンがJAPANに巣食う強大な魔族を斃しに赴いた地であるのだが、今回の君の目的は、それとは別の案件だ。

　『不老不死の妙薬の入手及び製造方法が記された書の入手』。
　これこそが君がペンタウァ国王に命じられ、このJAPANという国に赴いた目的である。

　だがしかし、あろうことか君は今、JAPANの片田舎で盛大に迷っていたのだ！
　元々が土地勘のない、更には文化も違う未知の国である。迷子になるのも仕方がないと言えば仕方がないことなのだが、それでもやはり祖国で『勇者していた』だけに、その落差と事実は君の疲労を増加させていた。

　この国での野宿にはもう慣れたものだが、そろそろ安全な場所で眠りたい。『フトン（布団）』と呼ばれる長方形の綿入り布袋のような寝具で寝たいなどという贅沢は言わないが、せめて人里の家屋に入って心身を落ち着ける時間が欲しい。
　……あともう少しだ。
　君は遥か前方に灯る温かな光の集合体を見つめ、重くなった脚に喝を入れて歩みを速めた。
　疲労でやや充血した君の視線の先にあるのは、小規模の人里だったのだ。
<br>
**村を目指して歩き続け――**
***
#### Scene 1
　ようやく人里に到着した君は、その入り口で見張りの男達に怪しまれてしまった。
　夕刻をとうに過ぎた頃に、JAPANでは見慣れない風体の人間が1人でヨタヨタと村の入り口に寄ってきたのだ。男達の対応は当然のことである。

　君は正直にペンタウァという西洋の国から船で渡ってきたソーサリアンという旅人だと男達に説明したが、男達は首を傾げ、ペンタウァとソーサリアンの発音もままならない様子だ。
　これまでの旅で実感していたことだが、ペンタウァでのソーサリアンの知名度は、JAPANではないに等しい。つまり、JAPANではペンタウァでの名声は一切通用しないと思った方がいいようだ。

　見張りの男達と君の問答は暫く続いた。
　JAPANESE MONSTER――すなわち東洋の化け物『ヨウカイ（妖怪）』と疑われもしたが、君が粘り強く説得を続け、切実に休憩場所を欲しているのだと訴え続けた結果、見張りの男に呼ばれてきたこの村の代表である男の屋敷に招かれることとなった。

**Comment by snext1220** [comment]
お返事ありがとうございます！

> シナリオを幾つか書きかけで紆余曲折し、ようやく和風シナリオにしようと決まり、現在執筆中です；

おぉ、幕末に先行して和風シナですか…！
導入を拝読しても雰囲気十分で、出来上がりが楽しみです～
10月中、についても承知いたしました。どうぞよろしくお願いいたします！

一ノ章ということで、連作的な?構造なのかな、と想定しておりますが、もし可能であれば連作機能使ってやって戴けますとm(_ _)m 結局、実装から一年、実践導入せず終いだったもので、幕末前に試験的にでもご利用いただけると、ゲストさん参加の前に課題を洗い出せるかなと（ないに越したことはないのですが、が、が；）。あくまで宜しければ、というレベルですが、ご検討くださいませ～

そして、私の方ですが、スミマセン、8月以降、PgFlowの改定で完全に手一杯状態で、シナリオに振り向けられておりません。。幸いちゃなさんの進行拝見する限り、10月末に公開できるのかなと勝手予想していたため、私の方はPgFlowの改善に振り切らせて戴こうかと（て、その後、幕末第一作目もあるので、早めにそちらへなんとか...！）。

もちろん、Pg100、ミニ30についても、幕末一作目のあと、振り向けていければと思いますので、恐縮ですが、お許しいただけますと。。シナリオ側はときのじさんらにおまかせっきりになってしまい、本当に申し訳ありません；；

**Comment by toki-sor1** [comment]
お返事が遅くなってしまい申し訳ありません…；

> おぉ、幕末に先行して和風シナですか…！
導入を拝読しても雰囲気十分で、出来上がりが楽しみです～
10月中、についても承知いたしました。どうぞよろしくお願いいたします！

お褒めのお言葉ありがとうございます！！
最初は洋風だったのですが、どうも真新しさがないなぁと思って、それなら逆に和風にしてみたらどうだろうと転換したら意外とシックリきて、和風シナリオ採用となりましたｗ
幕末ソーも控えていたので、ウォーミングアップの意味もありますねｗ

> 10月中、についても承知いたしました。どうぞよろしくお願いいたします！

当初は10月中に公開の予定でしたが、どうにも厳しそうなもので、申し訳ありませんが公開月を11月または来年にスライドさせていただいてもよろしいでしょうか？；；
幕末ソーのこともありますので、11月上旬までに公開できそうにない場合は、来年に回していただけると助かります…；

> 一ノ章ということで、連作的な?構造なのかな、と想定しておりますが、もし可能であれば連作機能使ってやって戴けますとm(_ _)m 

旅の目的を考えるとシリーズ化しても面白いかなと思って仮に一ノ章とつけておりましたので、連作対応も可能です～。
ただ、「ハクジャ伝説」が終わったら次の章へ、というイメージであることと、二ノ章の構想がまだ浮かんでいないことから、幕末ソー前に二ノ章は投稿できそうにないですね…申し訳ありません；；

> そして、私の方ですが、スミマセン、8月以降、PgFlowの改定で完全に手一杯状態で、シナリオに振り向けられておりません。。

> もちろん、Pg100、ミニ30についても、幕末一作目のあと、振り向けていければと思いますので、恐縮ですが、お許しいただけますと。。

いえいえ、怒涛の更新でしたもんね…！　本当にお疲れ様でございます…！！
リオさんが多くのご意見やご感想をもとにPgFlowの改善に全力を注いでくださったことで、PgFlowが劇的にパワーアップしてとても使いやすくなりました！　ありがとうございます…！！
「ソーサリアベンジャーズ」もめでたく公開となり、お次は幕末ソー（とPg100祭も…！）が控えておりますが、引き続きよろしくお願いいたします…！

**Comment by snext1220** [comment]
@toki-sor1  さん：

お返事ありがとうございます！

ウォーミングアップ＞ホントこれですよね～私の方もドラモードゼロ以来、専らシステム改定に感けて、シナリオは久々になってしまったので、微妙にウラシマさん状態です；全然調子（頭）が戻ってこずに、苦戦中です＞＜

> 公開月を11月または来年にスライドさせていただいてもよろしいでしょうか？；；幕末ソーのこともありますので、11月上旬までに公開できそうにない場合は、来年に回していただけると助かります…；

了解です！勿論、ご無理ない範囲でお願いします～
＃連作機能ももしうまく活用頂けるようであれば、という程度でしたので、無理せず使ってやってくださいませm(_ _)m

> リオさんが多くのご意見やご感想をもとにPgFlowの改善に全力を注いでくださったことで、PgFlowが劇的にパワーアップしてとても使いやすくなりました！

成果物が見えにくい作業も多く、役に立ってるのかな？と手探り状態だもので、そう仰って戴けると、救われます^^; 是非、新作作成の折にもお気づきの点があればなんなりとお声がけください！（ちゃなさんのソーサリアベンジャーズ進行は、まさにPg100祭でしたｗ）

ではでは、引き続きよろしくお願いいたします～

**Comment by snext1220** [comment]
https://github.com/snext1220/stext/issues/247#issuecomment-832384930 由来で投稿しておきます。
ミニ30、Pg100に続く自由枠企画を検討できればと思っています。当初意図としては、既存企画枠にはまらないものを～とのことだったので、以下のような受け皿になるのかなと。

+ スペクタクル300（仮名）
    + 300パラ以上の長編
    + ハクジャのような連作もの
    + その他に、間口を広げられそうな要素があれば

長編ということで、正直投稿のハードルは上がりますが、アベ、（場合によっては）ドラなどを移動しつつ、気長に充実させていければ良いなぁと（ただ、連作を許容することで、多少ハードルは下がるかなとも）。

ミニ30、Pg100、スペ300ができると、一通り短編、中編、長編の受け皿が揃うので、徐々に自由シナについても充実させていければ良いなぁと思っています。皆さんの今後の予定についてもお教え戴ければ幸いです（私の方は https://github.com/snext1220/stext/discussions/261#discussioncomment-667727 にて）。

**Reply by Salvadors-cabin** [reply]
どうも、サルバです。
最近公私ともに忙しく、なかなか思うように活動できていませんが、考えていることを書いておきます。

次の目標：
・ソーサリアン原作35周年 2022年12月20日

やりたい事：
(1) SN2020
 ① 追加シナリオ
　神剣No.2「怒りの追撃」になるかな・・
 ② システム更新
　EXP、GPを活用した機能追加
(2) SText
・長編シナリオ　※スペ300のカテゴリーの予定
　「ラミディア（ロマンシア）」世界観で構想中。

現在は(1)-②の下準備としてEXPの管理方法見直しを行っています。

**Reply by snext1220** [reply]
@Salvadors-cabin さん：

お忙しい処ありがとうございます！
早いもので原作も35周年ですか...そして同じく来年（2022年11月）はNextチーム発足10周年だったりもします（ 発足時期微妙だったりしますが、そんなものですよねｗ＞ @toki-sor1 さん）。
作品も徐々に追加していきたいのは勿論、コロナも落ち着いて、その頃にはオフ会もできるようになっていればよいですね。

閑話休題。SN2020、STextとありがとうございます。
そして、スペ300への投入予定との事も本当に（ry 長編となると、中々製作期間も長くなりますが、徐々に充実させていきたいですね。
＃ とりあえずスペ300の対象範囲も決めて、企画として立ち上げるところから。

そして、35周年のお話が先に出てしまいましたがｗ、34周年も軽く何かできればなどと。
STextは幕末ソーサリアンの只中なので、大きなものは35周年持ち越しかなとは思っておりますが、短編（ミニ30）などで12月のみ週刊ソーサリアンとか、50パラ×4人で200パラ合作シナなどもありかななど、勝手に思っています。こちらも是非ご意見いただければ幸いです～

**Reply by cocotori** [reply]
話題自体を見落としてしまっていたようで、申し訳ありません。

現状ではやりたいアイデアは固まっていないのですが・・・

・SN2020
せっかくのシステムがありますので、こちらには
なにか作品を投稿したいです。

・ソーサリアンText
正直なところ長編物を作れる自信がなく、
スペ300が開催された場合の参加は辞退させていただこうかな・・・と考えています。
代わりに、短めのシナリオでなにか参加できればと思っています。

**Reply by snext1220** [reply]
#### @cocotori さん：

> 短めのシナリオでなにか参加できればと思っています

はい、勿論大歓迎です！どうぞよろしくお願いいたします～

＃ スペ300は正直長編の受け皿がないのを受けて、と立ち上げようというレベルだったので、従来の企画とは異なり、「そちらの分類に適していれば追加していく」という程度で良いのかな、と考えています^^; 長期的には少しずつ増えていければ良いな、という程度です。

#### 皆さん：

そして、34周年についても、引き続き皆さんのご意見を伺えればと思いますので、どうぞよろしくお願いいたします。

今のところは、短中編枠で「日本昔話／童話ソーサリアン」などできないものかな、などと（ https://github.com/snext1220/stext/discussions/261#discussioncomment-667727 でちらっと触れたもの）。勿論、それがすべてではなく、**ドラモード／幕末程には制限を設けないが、ミニ30、Pg100よりはなんらかのテーマを設けられれば** くらいの緩いものを考えておりますので、ご意見など戴ければ幸いです！
＃ もちろん、完全自由枠でやりたい、というご意見もあるかとは。

**Comment by toki-sor1** [comment]
お返事大遅刻申し訳ありません……！

> 早いもので原作も35周年ですか...そして同じく来年（2022年11月）はNextチーム発足10周年だったりもします（ 発足時期微妙だったりしますが、そんなものですよねｗ＞ @toki-sor1 さん）。

そうですね、当時の素材作成日時を確認してみると、やはり2012年11月頃が発足時期っぽい感じがしますｗ
あの頃はまだ、チーム活動とか考えずにやっていましたし、発足時期が曖昧なのも仕方なしですよね……ｗ
いやぁ、来年でNextチーム発足10周年ですか……感慨深いですね……ｽﾞｽﾞｯ……（鼻すすり）

34周年についてですが、今のところは幕末ソー「参」の制作を優先しているので、今後の見通しはまだ立っていません；
ただ、**STextで「ハクジャ伝説」が途中になっているので、「参」後はそちらに着手すると思います。**

私もですが、チームの皆さんもそれぞれにご自身の創作活動もあると思いますし、ドラモードや幕末ソーレベルの大型企画が連続すると、体力と時間的になかなか厳しいものがありますね；

充電期間も必要かなと思いますので、個人的には**題材は決めずに、大まかなパラグラフ数のみ設定した自由に書ける緩めな企画**が良いのかなと考えています～。

**Reply by snext1220** [reply]
> いやぁ、来年でNextチーム発足10周年ですか……感慨深いですね……ｽﾞｽﾞｯ……

本当に感慨深いですよね...チームも考えてなかったくらいですから、まさか0401の後、10年経ってもまだ活動しているとは。今後ともどうぞよろしくお願いします！

> ただ、STextで「ハクジャ伝説」が途中になっているので、「参」後はそちらに着手すると思います。

ご予定ありがとうございます＆了解いたしました～
体力／時間、は仰る通り悩ましいですね；私も、ここのところ督促じみたご連絡が増えてしまっているのが良くないなぁと感じておりましたし。。とりあえずUpdateも主なものが捌けたので、一旦、隔月にするなど空けていった方が良いのかな、と考えております（ゲストさんリクエストの様子も見ながらとは；）。

**ご相談** サルバさんのラミディア続編？、ハクジャ（連作扱い？）なども出てきたところで、スペ300も長編／連作枠として、分類のみ作成しておければと思っておりますが、如何でしょうか（なんとなく反対のご意見もなかったようなので^^;）。全く急ぎではないのですが、落ち着いたところでバナーなどご相談させて戴けますと幸いでございますm(_ _)m

> 大まかなパラグラフ数のみ設定した自由に書ける緩めな企画が良いのかなと考えています～。

こちらも承知いたしました。
34周年の区切りのため、テーマが全くなしとなると、当日のインパクトはやや弱いかなと気にしておりましたが、、、充電期間とのご指摘も御尤もかと。ということであれば、皆さんのご意向次第では、34周年はスキップと致しましょうか。自由シナの展開は、34周年とは別に、それぞれ気の向くままに進めていくという感じで^^;

まだ34周年までは時間ありますので、またお手すきの折にご意見いただければ幸いです～

**Reply by toki-sor1** [reply]
> 本当に感慨深いですよね...チームも考えてなかったくらいですから、まさか0401の後、10年経ってもまだ活動しているとは。今後ともどうぞよろしくお願いします！

10年経っても活動が続いていることを考えると、皆さんとはまさに運命的な出逢いと言っても過言ではありませんよね～！
こちらこそ今後ともどうぞよろしくお願いします～！！

> とりあえずUpdateも主なものが捌けたので、一旦、隔月にするなど空けていった方が良いのかな、と考えております（ゲストさんリクエストの様子も見ながらとは；）。

はい～、企画自体も大きなものになってきましたし、作業や連絡等の時間が増えてしまうと、純粋な創作時間や自由時間が削れてしまい、その積み重ねが心身の負担へと繋がってしまうと思います；
今後長く創作活動を楽しむためにも、**一旦、隔月にしてみて様子を見るのも良いと思います。**

もちろん、**緊急性（優先度）が高い事柄については、隔月でも関係なくお話を振っていただいて、その時に対応する感じかな**と思っています～。

> スペ300も長編／連作枠として、分類のみ作成しておければと思っておりますが、如何でしょうか（なんとなく反対のご意見もなかったようなので^^;）。

こちらこそお返事が遅くなってしまい、どうもすみません；
はい、既に300パラグラフ越えのシナリオもありますから、**予めスペ300も長編／連作枠として、分類のみ作成するのもよいと思います！**

**バナーの件も承知しました！**
詳しいことが決まり次第、改めてお声を掛けていただけましたら助かります～。

> 皆さんのご意向次第では、34周年はスキップと致しましょうか。自由シナの展開は、34周年とは別に、それぞれ気の向くままに進めていくという感じで^^;

そうですね、ドラゴンモードや幕末ソーサリアンを既に展開しておりますから、記念企画としてテーマなしは、かなりインパクトが弱いですね；
となれば、**無理に34周年の企画にすることもないかな**という感じもします；
＃実際、0401企画からSTextまで数年は開いてますので、連続して企画発表ということにそこまで拘らなくてもよいのでは、とも

皆さんのご意向次第ではありますが、**34周年企画はスキップ。自由シナリオは、34周年とは別にそれぞれ気の向くままに進める**というのがよいかなと思います～。

**Reply by snext1220** [reply]
@toki-sor1 さん：
お忙しい処、お返事ありがとうございます。すべて了解です！
Monthly Updateについては当面隔月とし、その後様子を見ながら四半期に空けることも検討していければと（間で適宜緊急リリースはするかもしれませんが、あくまでテストは後追い）。また、いずれの場合も〆のタイミングで区切って、当方判断でCloseするなどして、皆さんヘのReminder的なご連絡を減らし、ご負担の軽減に努めていければと思っています。

スペ300についてもありがとうございます。
参が落ち着いたところで改めてまとめさせていただきますね。
では、今後ともよろしくお願いいたします～

**Reply by snext1220** [reply]
皆さま：

充電期間とのお話を戴いてから、現在のUpdateについても諸々考えていたのですが、今後、以下のようにできないかと考えております。

#### Quarterly Update

Monthly UpdateからQuarterly Updateに移行し、間を空けられればと思います。
大雑把には「2021 Summer Update」などのような名前になっていくのかなと。

#### Reminderはなくします

Update 〆のタイミングで適宜私の方の判断でClose、または次回Updateに持ち越してまいります（Ｄなどでの確認は割愛）。

#### 本番反映は随時

Update（動作テスト）の間隔が開くに伴い、個別の改定／バグは適宜Issueを立ち上げつつ、作業を進め、本番環境にも随時反映していければと思います（非互換を伴う改定はもちろん一定ご意見待ちの期間を設けて、慎重に行っていく予定です）。

如何でしょうか。連絡系統が私も含めて負担の原因にもなっていたと思われるので、その辺軽減しつつ、改定の足回りは維持していく、という案です。System側の裁量にある程度お任せいただくことにはなりますが、動作テストがなくなるわけではないですし、もちろん、リクエスト／バグ報告にはこれまで通り対応していきますので、お気軽にお声がけください。

＃ 同じく企画ものの立ち上げも、タイミングなどはかるのがなんとも悩ましく、今後どうしていこうか思案しているのですが、こちらはおいおいご相談させてくださいませ。

**Reply by toki-sor1** [reply]
ご提案ありがとうございます！

> Quarterly Update

「2021 Summer Update」など、年4回に区切るのですね。
STextも皆様のご協力により機能がとても充実し、新機能を提案することの方が難しくなってきたため、年4回の区切りも良いかなと思います。

> Reminderはなくします

こちらも連絡系統の負担軽減と、意見待ちによる時間のロスを削減できるため、良いと思います。

> 本番反映は随時

はい、こちらのご方針も異論ありません。

> 連絡系統が私も含めて負担の原因にもなっていたと思われるので、その辺軽減しつつ、改定の足回りは維持していく、という案です。～

私の方は、ご提案いただいたご方針に全面同意です！
連絡系統はどうしても負担になりがちですし、全ての事柄を意見が揃い次第まとめてUpdateするよりも、影響が少ない内容については随時Updateできた方がスムーズで良いかなと思います。
動作テスト、リクエストやバグ報告等は、引き続き行ってまいりますので、よろしくお願いします！

> ＃ 同じく企画ものの立ち上げも、タイミングなどはかるのがなんとも悩ましく、今後どうしていこうか思案しているのですが、こちらはおいおいご相談させてくださいませ。

はい～、承知しました！
大きな企画になると、年単位の準備期間が必要になってきたりしますもんね；
また、考えてみればドラゴンモードも幕末ソーサリアンもSN2020も、構想から企画立ち上げまでに時間を要しており、現在ではこれらの企画が完結or進行中なので、今は大きな企画のストックが切れた状態とも言えます；

英気を養いつつ、次の企画も考えていかねばですね～……！

**Reply by snext1220** [reply]
諸々ご賛同ありがとうございます！
他の皆さまのご意見もお待ちいたしますが、まずはその方向で進めてまいりますね。
先ほどWikiも全体更新しましたので、お手すきの折にご確認ください。
https://github.com/snext1220/stext/wiki/%E6%A9%9F%E8%83%BD%E3%83%86%E3%82%B9%E3%83%88%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6

> 連絡系統はどうしても負担になりがちですし、全ての事柄を意見が揃い次第まとめてUpdateするよりも、影響が少ない内容については随時Updateできた方がスムーズで良いかなと思います。動作テスト、リクエストやバグ報告等は、引き続き行ってまいりますので、よろしくお願いします！

はい、こちらこそよろしくお願いします～懸念は見落とし＆実装後の手戻りですが、たまにIssueも覗いてやって戴けると幸いです^^;

> 現在ではこれらの企画が完結or進行中なので、今は大きな企画のストックが切れた状態とも言えます；英気を養いつつ、次の企画も考えていかねばですね～……！

はい、企画立ち上げは時間も気力も要するので；、ゆるゆるとでも早めに考えていきたいですね…！参が落ち着いたところでまたお声がけくださいませ～

**Reply by cocotori** [reply]
いつも確認が遅れてしまう身なので本当に申し訳なく感じますが、
負荷軽減の方向に舵取るのに賛成です。

・Quarterly Update
・Reminderはなくします
・本番反映は随時

ご提案の方策、どれも賛成です。
また、新しいシステムを立ち上げたときなどは
頻繁なアップデートが必要になりやり方を見直す必要があるかと思いますが、
STextのように基本機能は完成し、少しずつ拡張していく段階のものは
そのくらいの更新頻度がよいのではと思います。

**Reply by Salvadors-cabin** [reply]
サルバです。

運用変更について私も異論ありません。
これまでもできる範囲でやってきましたので、及ばずながら・・・ではありますが。


>そうですね、当時の素材作成日時を確認してみると、やはり2012年11月頃が発足時期っぽい感じがしますｗ

調べてみたら私がリオさんからメール連絡いただいたのは、2012年10月でした。
たぶんその頃にはシナリオのWeb掲載など、始まっていたんですよね。
その後、企画にお誘いいただき、旧掲示板へ私が最初の書き込んだのは2013年1月だったようです。
ホント、あっという間ですね・・。

来年はワクチン接種も完了し、元の世界に戻ってほしいですね・・。
そして、またみなさんとオフ会開催したいです！

>・SN2020
>せっかくのシステムがありますので、こちらには
>なにか作品を投稿したいです

@cocotori 　さん
大変ありがたいです！
私の方でもコツコツ修正しているので、具体的になりましたらお声掛けくださいー。

**Reply by snext1220** [reply]
@cocotori さん、 @Salvadors-cabin さん：
ご賛同ありがとうございます。
では、これで皆さんの同意も得られたので、新しい形で進めてまいりますね。
とりあえず最初のSummer Update（https://github.com/snext1220/stext/milestone/30 ）は7月下旬には確定し、8/7〆となる予定です～

頻繁なアップデートが必要になりやり方を見直す必要が～＞はい、特に大きな企画の前や、ゲストさんからのリクエストが増えてきた場合には、不定期Updateなどご相談させて戴けますと幸いです。

> 調べてみたら私がリオさんからメール連絡いただいたのは、2012年10月でした。

はい、シナコンの結果発表からジワジワ応募作品の公開をしていたのですが、いきなりの打診に快諾戴いて、改めてありがとうございました^^;
応募作品の公開が一段落して、「あれ、この15（14）シナ並べてみたら、Nextになるんでは？」と始めた感じだったので、やはり11月位が最初になりそうですね。

10周年オフ会に向けて?、引き続きよろしくお願いいたします！

**Reply by Salvadors-cabin** [reply]
サルバです。
SN2020システムの機能追加を行っています。
O/Dに追加機能の仕様書とサンプルプロジェクトをアップしましたのでお時間のあるときにでも見ていただけると幸いです。
https://onedrive.live.com/?authkey=%21As9FZ%2D7maNlcx70&id=4837C202555BE782%2110720&cid=4837C202555BE782
仕様書：追加機能「町へ行く」.pptx



なお、最近DMで話題になりましたグレーゾーンについての考えは企画立ち上げ時の企画書の「権利関係」のページを参照してください。
https://onedrive.live.com/?authkey=%21As9FZ%2D7maNlcx70&id=4837C202555BE782%219239&cid=4837C202555BE782
今回の機能追加でもスタンスに変わりはありません。
いっそう原作寄りのシステムになるわけですが、原作の画像・音源・コード等の利用はありませんし、細部は変更しており、全くそのままではありません。
一般的な自主制作ゲーム（オマージュゲーム）の範囲と思っています。
